app/services/daily_usages/fill_from_invoice_service.rb

```diff
@@ -55,7 +55,7 @@ def invoice_usage(subscription, invoice_subscription)
         amount_cents: invoice.fees_amount_cents,
         total_amount_cents: invoice.fees.sum(&:total_amount_cents),
         taxes_amount_cents: invoice.fees.sum(:taxes_amount_cents),
-        fees: invoice.fees.charge.select { |f| f.subscription_id == subscription.id }
+        fees: invoice.fees.charge.select { |f| f.subscription_id == subscription.id }.select { |f| f.units.positive? }
```

Comment:
```suggestion
        fees: invoice.fees.charge.select { |f| f.subscription_id == subscription.id && f.units.positive? }
```
Nitpick since the number of fees should be very small. Feel free to keep the second select for clarity/consistency.